extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -132,6 +130,16 @@ function () {
 					];
 				} );
 
+				foreach ( $validAdminIds as $adminId ) {
+					if( !in_array( $adminId, array_column( $result,  'userId' ) ) ){
```

Comment:
What do you think about preparing this all admins array with default values ie 
```
$validAdminIds = [
    adminId = [
        'userId' => adminId,
        'contributions' => 0,
        'isAdmin' => true
    ],
    ...
]
```
and then in `$sqlData->runLoop` only update `contributions` ?